repo/packages/C/confluent/3/config.json

```diff
@@ -0,0 +1,571 @@
+            "type":"integer",
+            "default":600000
+          },
+          "controlled_shutdown_enable":{
+            "title":"controlled.shutdown.enable",
+            "description":"Enable controlled shutdown of the server",
+            "type":"boolean",
+            "default":true
+          },
+          "controlled_shutdown_max_retries":{
+            "title":"controlled.shutdown.max.retries",
+            "description":"Controlled shutdown can fail for multiple reasons. This determines the number of retries when such failure happens",
+            "type":"integer",
+            "default":3
+          },
+          "controlled_shutdown_retry_backoff_ms":{
+            "title":"controlled.shutdown.retry.backoff.ms",
+            "description":"Before each retry, the system needs time to recover from the state that caused the previous failure (Controller fail over, replica lag etc). This config determines the amount of time to wait before retrying.",
+            "type":"integer",
+            "default":5000
+          },
+          "controller_socket_timeout_ms":{
+            "title":"controller.socket.timeout.ms",
+            "description":"The socket timeout for controller-to-broker channels",
+            "type":"integer",
+            "default":30000
+          },
+          "default_replication_factor":{
+            "title":"default.replication.factor",
+            "description":"Default replication factors for automatically created topics",
+            "type":"integer",
+            "default":1
+          },
+          "fetch_purgatory_purge_interval_requests":{
+            "title":"fetch.purgatory.purge.interval.requests",
+            "description":"The purge interval (in number of requests) of the fetch request purgatory",
+            "type":"integer",
+            "default":1000
+          },
+          "group_max_session_timeout_ms":{
+            "title":"group.max.session.timeout.ms",
+            "description":"The maximum allowed session timeout for registered consumers",
+            "type":"integer",
+            "default":30000
+          },
+          "group_min_session_timeout_ms":{
+            "title":"group.min.session.timeout.ms",
+            "description":"The minimum allowed session timeout for registered consumers",
+            "type":"integer",
+            "default":6000
+          },
+          "inter_broker_protocol_version":{
+            "type":"string",
+            "title":"inter.broker.protocol.version",
+            "description":"Specify which version of the inter-broker protocol will be used. This is typically bumped after all brokers were upgraded to a new version. Example of some valid values are: 0.8.0, 0.8.1, 0.8.1.1, 0.8.2, 0.9.0.0, 0.9.0.1, 0.10.0.0  Check ApiVersion for the full list.",
+            "default":"0.10.0.0"
+          },
+          "log_cleaner_backoff_ms":{

```

Comment:
Could you insert this new parameter here? Looks like it would be useful for 0.10.0, and I just added support for passing it through in the latest version of the scheduler:

```
          "log_message_format_version":{
            "type":"string",
            "title":"log.message.format.version",
            "description":"Specify which version of the log message format will be used, which must align with inter.broker.protocol.version. This is a new setting as of 0.10.0.0, and should be left at 0.9.0 until clients are updated to 0.10.0.x. Clients on earlier versions may see a performance penalty if this is increased before they've upgraded. See the latest Kafka documentation for details.",
            "default":"0.10.0"
          },
```

(see https://github.com/mesosphere/universe-1.7/blob/version-2.x/repo/packages/K/kafka/15/config.json#L424 )